src/ui/src/routes.ts

```diff
@@ -1,13 +1,22 @@
-import { IRoute } from "./interfaces/IRoute";
-import Home from "./pages/Home/Home";
+import { IRoute } from './interfaces/IRoute';
+import Home from './pages/Home/Home';
+import Welcome from './pages/Welcome/Welcome';
 
-export const routes: Array<IRoute> = [
-    {
-        key: 'home-route',
-        title: 'Home',
-        path: '/',
-        enabled: true,
-        component: Home
-    },
-  
-]
\ No newline at end of file
+const routes: IRoute[] = [
+  {
+    key: 'home-route',
+    title: 'Home',
+    path: '/home',
+    enabled: true,
+    component: Home,
+  },
+  {
+    key: 'welcome-route',
+    title: 'Welcome',
+    path: '/',
+    enabled: true,
+    component: Welcome,
```

Comment:
I think '-route' part for keys is redundant since we definitely know that these are. Remove empty space between 14 and 15 lines